### DISCUSSION TITLE
Compiling code in memory

### DISCUSSION THREAD

**Discussion by dusx1981** [facebookincubator/velox#5840] [Ideas]
why not perform compiling and linking operations in memory instead of writing to files, which adds IO operations? Will this have a significant impact on performance?

**Comment by mbasmanova** [comment]
CC: @laithsakka

**Comment by laithsakka** [comment]
can you explain more, are you asking about
experimental/codegen?

**Comment by laithsakka** [comment]
if its about codegen, we were targeting large queries that takes a longer time, so compilation and linking cost was not an issue.
